Simplify Conda Install, Create Offline Install

milljm opened the pull request:
Create a new offline install markdown page.

Simplify Conda install by only supplying commands to install Miniconda3 via bash.
Split up the miniconda install page, so we can use it, without expressively installing
moose.
Fix mention of moose-environment in ccache help.

Closes #15436 #15441 #15414

Commit:
Simplify Conda Install, Create Offline Install

Create a new offline install markdown page.

Simplify Conda install by only supplying commands to install Miniconda3 via bash.
Split up the miniconda install page, so we can use it, without expressively installing
moose.

Closes #15436 #15441

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/544429/) on a9d1736 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/15443/site)

This comment will be updated on new commits.

Commit:
Add environment setup help

Before building PETSc offer additional advice on setting up the environment.

Closes #15414

Commit:
Fix PATHs before building PETSc

Fix some PATH instructions.
Alter the wordage with tar compress/extract, and to where.

Closes #15414

rwcarlsen reviewed (approved):
Not sure if anyone else wanted to take a look too - so I'll let someone else do the merge honors.  But this looks good to me.

cticenhour reviewed (approved):
Looks good to me as well.

jessecarterMOOSE commented:
:+1: for offline builds
